orchestra/utils/notifications.py

```diff
@@ -83,11 +85,21 @@ def notify_status_change(task, previous_status=None):
                                assignment.worker.user.email]
         }
 
-    _notify_internal_slack_status_change(task, current_worker)
-    if task.project.slack_group_id:
-        _notify_experts_slack_status_change(task, current_worker)
-
-    if message_info is not None:
+    # TODO(joshblum): Find a good abstraction for checking preferences
+    comm_type = CommunicationPreference.CommunicationType.TASK_STATUS_CHANGE
+    comm_pref = get_object_or_None(
```

Comment:
can this return two objects? What happens if slack and email are both set for TASK_STATUS_CHANGE?